libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1822,39 +1821,6 @@ void GCS_MAVLINK::send_scaled_pressure3()
     send_scaled_pressure_instance(2, mavlink_msg_scaled_pressure3_send);
 }
 
-void GCS_MAVLINK::send_sensor_offsets()
-{
-    const AP_InertialSensor &ins = AP::ins();
-    const Compass &compass = AP::compass();
-
-    // run this message at a much lower rate - otherwise it
-    // pointlessly wastes quite a lot of bandwidth
-    if (send_sensor_offsets_counter++ < 10) {
-        return;
-    }
-    send_sensor_offsets_counter = 0;
-
-    const Vector3f &mag_offsets = compass.get_offsets(0);
-    const Vector3f &accel_offsets = ins.get_accel_offsets(0);
-    const Vector3f &gyro_offsets = ins.get_gyro_offsets(0);
-
-    const AP_Baro &barometer = AP::baro();
-
-    mavlink_msg_sensor_offsets_send(chan,
-                                    mag_offsets.x,
-                                    mag_offsets.y,
-                                    mag_offsets.z,
-                                    compass.get_declination(),
-                                    barometer.get_pressure(),
-                                    barometer.get_temperature()*100,
```

Comment:
This isn't actually available via the parameters, but it is in the scaled pressure, which is better anyways.